src/corelibs/U2Core/src/util/IOAdapterUtils.h

```diff
@@ -33,10 +33,10 @@ class U2CORE_EXPORT IOAdapterUtils {
 public:
     static IOAdapterId url2io(const GUrl &url);
 
-    static QByteArray readFileHeader(const GUrl &url, int size = 65536);
+    static QByteArray readFileHeader(const GUrl &url, int size = 524228);
```

Comment:
Create the `static constexpr` variable with the meaningful name - now it'n not clear what is it about

Reply:
IMHO No reason to do that it look like tautology. 
Because everybody know what is 'default parameter' in C++ is.
Also im not sure is this construction possible for compiler.
For me is look like:
X = cos(t)
t = 30°
Instead of :
X = cos(30°)

Reply:
I don't like this unknown parameter, it's not obvious where it's got from. Add at least a comment